pkg/operator/ceph/config/config.go

```diff
@@ -138,6 +138,17 @@ func SetOrRemoveDefaultConfigs(
 		}
 	}
 
+	// remove `rbd_default_features` since it is no longer used in the
+	// upgraded clusters. This was set to `3` previously from Rook.
+	rbdDefaultFeatures := "rbd_default_features"
+	val, err := monStore.Get("mon", rbdDefaultFeatures)
+	if err == nil && val == "3" {
+		logger.Infof("removing rbd_default_features config")
```

Comment:
Debug log? Especially if we go with a Delete only?

Reply:
Will wait for the above Get/Delete comment based on that i will address this one :+1: